pandas/tests/io/excel/test_readers.py

```diff
@@ -38,13 +49,17 @@ class TestReaders:
         # Add any engines to test here
         pytest.param('xlrd', marks=pytest.mark.skipif(
             not td.safe_import("xlrd"), reason="no xlrd")),
+        pytest.param('openpyxl', marks=pytest.mark.skipif(
+            not td.safe_import("openpyxl"), reason="no openpyxl")),
         pytest.param(None, marks=pytest.mark.skipif(
             not td.safe_import("xlrd"), reason="no xlrd")),
     ])
-    def cd_and_set_engine(self, request, datapath, monkeypatch):
+    def cd_and_set_engine(self, request, datapath, monkeypatch, read_ext):
         """
         Change directory and set engine for read_excel calls.
         """
+        if request.param == 'openpyxl' and read_ext == '.xls':
```

Comment:
This should work for now. @simonjayhawkins something we will have to clean up when we get `.odf` files in the mix as well